### ISSUE TITLE
On summary panel, I have to scroll left in the gene table to see which phenolyzer conditions matched.

### ISSUE THREAD

**Issue by tonydisera** [iobio/genepanel.iobio#304]
<img width="1140" alt="screen shot 2018-11-14 at 9 38 35 am" src="https://user-images.githubusercontent.com/8420103/48497348-3230e580-e7f1-11e8-8c43-4751ad308490.png">

**Comment by adityaekawade** [comment]
This depends on the screen size. If the user is viewing it on a bigger screen, they should be able to see all the columns without scrolling right.
![Screen Shot 2018-11-14 at 2.29.13 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5a68ffc37453d30014ef4f0d/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b76226455f76b7191a37af110224f5e5c5ecc3c4a4356bcbbd671f3e57862804e0ebcd8380b7dcf6f14faee824dba01ba21d94d86e7b48818d2f42f67b6d5db1063474f4dcbed5727a19141c95df249e34e54a41b326be57f2431.png)

The container of the table becomes responsive based on the screen size and the content inside it tends to add a scroll. 
What I can do is rename some of the column names so that it takes less space. 
For example : "Gene Name" can be "Gene"

**Comment by tonydisera** [comment]
Yes, but I think that on the laptops, the resolution I show above is more typical.
